script/Arcueid/Day 9/09_00_ARC09_4B.txt

```diff
@@ -253,7 +253,7 @@ Akiha speaks in a reserved tone.
 [sha:45d5e0ccd2d53600aa3990959b71416dc4db12e0]{
 -- Page 213, Offset 21890.
 -- が、こっちの心配事は自分の体の事ではなくて、アルクェイドとの約束だった。
-However, I'm not worried about my body so much as I am the promise I made with Arcueid.
+However, I'm not worried about my body so much as I am of the promise I made with Arcueid.
```

Comment:
'of' does not feel correct here

Reply:
if you want to be explicit, 'about' is the word to be used here, but it's implicit from prior usage, and I don't really think it adds much in this case to be explicit